.github/workflows/bumpversion.yaml

```diff
@@ -7,27 +7,64 @@ on:
 
 jobs:
   bump_version:
-    if: "!startsWith(github.event.head_commit.message, 'bump:')"
     runs-on: ubuntu-latest
     name: "Bump version and create changelog with commitizen"
+    env:
+      GIT_USER_NAME: github-actions
+      GIT_USER_EMAIL: github-actions@github.com
```

Comment:
Are these valid for our purposes? I assume `git` would accept the values just fine, but I'm wondering if this is a good practice. Would we want to use a values that actually belong to us instead of GitHub, like an email/group name/alias for discovery/quipucords?

Reply:
I was aiming for something indicating the commit was made by a bot.
Using an user from our org definitely works as well =D

(any thoughts on quipucords-bot / quipucords@redhat.com ?)

Reply:
Do we already control the quipucords@redhat.com email address? If so, I think that's great. 👍 If not, we could probably acquire it easily enough…

Reply:
@gtanzillo has some control over it. I think you can get emails from that account.